test/test.js

```diff
@@ -188,6 +188,46 @@ test('errors', t => {
 		htmlMinifier
 	}), SyntaxError);
 	t.is(error.message.split(/[\r\n]+/)[0], filename + ': ' + plugin.majorDeleteError);
+
+	const commentedBindings = `
+		import {html} from 'lit-html';
+		const template = html\`
+			<div>\${a}</div>
+			<!-- <div>\${b}</div> -->
+		\`;
+	`;
+
+	const originalLog = console.error;
+	let loggedMessage = null;
+	console.error = msg => {
```

Comment:
I don't think it's actually possible to race but ava can run multiple tests simultaneously.  Just to be future proof/protect against potential async usage in this test we should mark this test serial.  Do this by updating the test declaration to `test.serial('errors', t => {`.  This will tell ava that other tests must not run at the same time.